tox.ini

```diff
@@ -4,10 +4,11 @@ envlist = py34, py33, py32, py27, py27-configparser, pypy
 [testenv]
 passenv = HOME
 deps=
+  distribute
   pytest
   mock
 commands=
-  py.test [] tests.py
+  py.test [] tests.py tests
```

Comment:
The smaller and the more focused on one aspect the better (a rule I live by for both for commits/pull requests). Don't worry about form, I'll split it up as needed. Me just needing to click the merge button might get the code in faster though.